### ISSUE TITLE
Firestore generation broken in versions >= 0.26.6

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#521]
The pagination fix in https://github.com/googleapis/gapic-generator-python/pull/517 seems to be the cause.

CC @crwilcox 

```
Traceback (most recent call last):
  File "/usr/local/bin/protoc-gen-python_gapic", line 8, in <module>
    sys.exit(generate())
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/gapic/cli/generate.py", line 58, in generate
    api_schema = api.API.build(req.proto_file, opts=opts, package=package)
  File "/usr/local/lib/python3.7/site-packages/gapic/schema/api.py", line 277, in build
    for name, proto in pre_protos.items()
  File "/usr/local/lib/python3.7/site-packages/gapic/schema/api.py", line 277, in <dictcomp>
    for name, proto in pre_protos.items()
  File "/usr/local/lib/python3.7/site-packages/gapic/schema/api.py", line 87, in build
    load_services=load_services
  File "/usr/local/lib/python3.7/site-packages/gapic/schema/api.py", line 511, in proto
    for k, v in naive.services.items()
  File "/usr/local/lib/python3.7/site-packages/gapic/schema/api.py", line 511, in <genexpr>
    for k, v in naive.services.items()
  File "/usr/local/lib/python3.7/site-packages/gapic/schema/wrappers.py", line 845, in __getattr__
    return getattr(self.service_pb, name)
AttributeError: names
--python_gapic_out: protoc-gen-python_gapic: Plugin failed with status code 1.
```

**Comment by software-dov** [comment]
Reproduced and verified that reverting #517 fixes the crash

**Comment by software-dov** [comment]
Fixed by #527
